src/api-docs.js

```diff
@@ -56,6 +56,10 @@ const apiDoc = {
                     type: 'boolean',
                     description: 'Start as a multisig wallet. Requires multisig configuration.'
                   },
+                  multisigKey: {
+                    type: 'string',
+                    description: 'Key of the multisig wallet data in the config. This allow wallets to be started without a seedKey, e.g. with the seed on the parameters or from an xpubkey.',
+                  },
                 }
```

Comment:
It's missing the new `xpubkey` parameter here, right?

Reply:
Added